### DISCUSSION TITLE
Open DTU & Herf 1600

### DISCUSSION THREAD

**Discussion by cha-ly** [tbnobody/OpenDTU#2155] [General]
Any there any open DTU user with a Herf 1600 inverter.

Here communication with the Herf 1600 is fine, 
I have an issue with the Herf AC power - either the inverter produces too low AC output (´defect?) or the conversion is inaccurate. 
I use 2x 550Wp (MPPT1 & MPPT2) but see only peak AC power level under 50W. 
Connecting one of the 550Wp panels to a Hoymiles HM600 I  see a reasonable of 350W (AC) Panels are 270Deg, pure West, ~ 45 Deg elevation.  
Dead inverter with low AC output???

**Comment by stefan123t** [comment]
@cha-ly did you measure the 50w AC output using a clamp multimeter or something similar in the AC line or is this the value from OpenDTU only ?

**Comment by cha-ly** [comment]
@stefan123t  Thanks for your hints. Yes measurements confirmed the low output. 
Sice  I have reset the Herf 1600 inverter to its default values three times in a row and left it off for some minutes. 

Back online the nominal output power was back.
May be the internal microprocessor hang?    
Even though the inverter is in a (screened) metal housing it may not be immune towards high local RF fields.. 
I am active radioamateur and I have seen a lot of electrical equipment to suffer from poor immunity.  

DC strings of PV Inverter often  pick up RF signals like  an antennae in particular if the line gets resonant.   
As a precaution I will add some clap Ferrites on the DC inputs.  
The string voltage is quite low, (under 50V) so adding a simple DC filter should cure the issue if it happens again.   
thanks 
Karl
